pkg/apis/stats/types.go

```diff
@@ -65,6 +65,38 @@ type AntreaNetworkPolicyStatsList struct {
 	Items []AntreaNetworkPolicyStats
 }
 
+// PodReference represents a Pod Reference.
+type PodReference struct {
+	// The name of this Pod.
+	Name string
+	// The Namespace of this Pod.
+	Namespace string
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// MulticastGroupMember contains the mapping between Multicast Group and Pods.
+type MulticastGroupMember struct {
+	metav1.TypeMeta
+	metav1.ObjectMeta
```

Comment:
If we use standard K8s API for this, what would be the name of the object?

Reply:
I think the the name of the object would be same as group IP

Reply:
Also think it is not necessary to have a single field for name also namespace.